src/ucp/rma/rma_sw.c

```diff
@@ -160,8 +163,10 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rma_cmpl_handler, (arg, data, length, am_flag
 {
     ucp_cmpl_hdr_t *putackh = data;
     ucp_worker_h worker     = arg;
-    ucp_ep_h ep             = ucp_worker_get_ep_by_id(worker, putackh->ep_id);
+    ucp_ep_h ep;
 
+    ep = UCP_WORKER_GET_EP_BY_ID(worker, putackh->ep_id, 0, return UCS_OK,
```

Comment:
why no check for closed ep?
here and in other places

Reply:
we could complete the operation, since the operation is completed
there are tests that expect it

Reply:
why tests expect operation completion on already failed/closed ep?